01_easy_algorithms/03_sort_words_by_length/03_sort_words_by_length.rb

```diff
@@ -0,0 +1,5 @@
+def sort_by_length(words)
+  arr_of_words = words.split(" ")
```

Comment:
Use single quotes unless you're using string interpolation or special symbols.
https://github.com/rubocop/ruby-style-guide#consistent-string-literals-single-quote